zerver/webhooks/front/doc.md

```diff
@@ -0,0 +1,37 @@
+Front lets you manage all of your communication channels in one place,
+and helps your team collaborate around every message. Follow these steps
+to recieve Front notifications without leaving Zulip!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Then, log into your Front account, and do the following:
+
+1. Go to the **Settings** of your organization.
+   ![](/static/images/integrations/front/001.png)
+
+2. Enable webhook notifications.
+   ![](/static/images/integrations/front/002.png)
+   ![](/static/images/integrations/front/003.png)
+
+3. Pick the types of events for which you want to receive notifications,
+and set the URL you construsted for the Front bot as a target webhook.
+   ![](/static/images/integrations/front/004.png)
+   ![](/static/images/integrations/front/005.png)
+
```

Comment:
Okay, so a couple of comments:
1. We don't need this many screenshots. In fact, if the process isn't too complicated, we shouldn't need any screenshots at all. See the discussion on #8433 to see what we are aiming for. The language should be standardised and consistent enough to point the user in the right direction (click on X in the top-left corner as an example). Unless something is super hard to distinguish in the third-party UI, we would like to refrain from using screenshots. UIs change around a lot!
2. This should ideally be structured as numbered steps (including the macros). Again see how I documented the integrations in #8433 for an example.